web/src/components/apps/EmbeddedClusterManagement.tsx

```diff
@@ -360,6 +362,50 @@ const EmbeddedClusterManagement = ({
   }, [nodesData?.nodes?.toString()]);
   // #endregion
 
+  const onContinueClick = async () => {
+    const res = await fetch(
+      `${process.env.API_ENDPOINT}/embedded-cluster/confirm`,
```

Comment:
this route name seems ambiguous or generic. not sure what's a better name though...

Reply:
`/embedded-cluster/management`?